src/domain/enrolment/EditEnrolmentPage.tsx

```diff
@@ -92,6 +92,12 @@ const EditorEnrolmentPage: React.FC = () => {
   React.useEffect(() => {
     if (enrolmentData) {
       const enrolment = enrolmentData.enrolment;
+      const studyLevels: string[] | undefined = enrolment?.studyGroup
+        ?.studyLevels?.edges?.length
+        ? enrolment.studyGroup.studyLevels.edges
+            .map((edge) => edge?.node?.id || '')
+            .filter((n) => n !== '')
+        : [];
```

Comment:
API describes the id mandatory and unique:
![image](https://user-images.githubusercontent.com/389204/106567017-cbb35180-6539-11eb-9657-602374c7b1aa.png)

I'm wondering if it is so because the edges and the node can be undefined.

Reply:
Ok I think it shows undefined just because of optional chhaining of course. I think in the type id is not optional